### DISCUSSION TITLE
V3.0.0 Faster, Stronger

### DISCUSSION THREAD

**Discussion by painty** [painty/CSS-Used-ChromeExt#53] [General]
- Migrated to Manifest V3.
- Resources are read from local caches, making it much faster and solves the https://github.com/painty/CSS-Used-ChromeExt/issues/52.
- Optimized style parsing in a more efficient way.
- Dropped support for some outdated CSS, mostly `-o-` and `-ms-` prefixed properties.
- Implemented a new UI powered by Svelte, with some visual tweaks.

<hr /><em>This discussion was created from the release <a href='https://github.com/painty/CSS-Used-ChromeExt/releases/tag/v3.0.0'>Faster, Stronger</a>.</em>

**Comment by realanthonyc** [comment]
Nice work! Thank you!
